src/main/java/org/broadinstitute/hellbender/tools/funcotator/filtrationRules/FuncotationFiltrationRule.java

```diff
@@ -0,0 +1,14 @@
+package org.broadinstitute.hellbender.tools.funcotator.filtrationRules;
+
+import java.util.Map;
+
+/**
+ * A rule to match against the Funcotations from a variant within a {@link FuncotationFilter}.
+ */
+interface FuncotationFiltrationRule {
+
+    /**
+     * Check if a set of Funcotations matches this rule.
+     */
+    boolean checkRule(final Map<String, String> funcotations);
```

Comment:
This is basically `Predicate<Map<String, String>>` now... worth refactoring to just use that type, or is the custom naming helpful for understanding?

Reply:
Yeah - I say leave it in for now.